maven-jxr-plugin/src/main/java/org/apache/maven/plugin/jxr/AbstractJxrReport.java

```diff
@@ -156,7 +156,7 @@
     private boolean linkJavadoc;
 
     /**
-     * Version of the Javadoc templates to use, ex. "4", "5" etc.
+     * Version of the Javadoc templates to use, ex. "1.4", "1.5" etc.
```

Comment:
Let's be more explicit here.  The value should reflect `java.specification.version`, "1.4", "1.8", "9", "10", by default this system property is used.